README.md

```diff
@@ -1,4 +1,140 @@
-# MicroCluster
+# Microcluster
 ## Introduction
-MicroCluster is a helper package for bootstrapping and managing a dqlite cluster.
-Based on [go-dqlite](https://github.com/canonical/go-dqlite).
+
+Microcluster is a Go library that provides an easy to use framework for creating and managing highly available [dqlite](https://github.com/canonical/dqlite) clusters. It provides an extensible API and database which can be used to directly connect to your underlying services.
+
+Extend Microcluster with your own API endpoints, schema and API extensions, lifecycle actions, and additional network listeners.
```

Comment:
"Extend Microcluster" sounds too gory; Microcluster gives me all of these tools for building my app. Take or leave:
```suggestion
Build your app with MicroCluster-managed HTTP endpoints, API extensions, database schema updates, lifecycle actions, and additional HTTP listeners.
```
Could also link to the structures/API for each of these things in the go doc.